src/stories/Blocks/event-list-page/EventListPage.tsx

```diff
@@ -1,11 +1,30 @@
 import React from "react";
 import EventList from "../../Library/event-list/EventList";
 import eventListData from "../../Library/event-list/EventListData";
+import { InputLabel } from "../../Library/input-label/InputLabel";
+import { Dropdown } from "../../Library/dropdown/Dropdown";
+
+const list = [
+  {
+    title: "Bob Dylan 80 år",
+    href: "/",
+  },
+  {
+    title: "5 Nordiske fuldtræffere",
+    href: "/",
+  },
+];
 
 const EventListPage: React.FC = () => {
   return (
     <div className="event-list-page">
       <h1 className="event-list-page__heading">Arrangementer</h1>
+      <div className="event-list-page__filters">
+        <div className="event-list-page__filter">
+          <InputLabel text="Kategorier" />
+          <Dropdown list={list} ariaLabel="Kategorier" arrowIcon="chevron" />
```

Comment:
Please reconsider this construct.

As I see it the label value is duplicated between the label component and the Dropdown component.

If we have an explicit label I would prefer that we had an explicit link between the two elements using `for` on the `<label>` or [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) on the form element.